helm/csi-vxflexos/values.yaml

```diff
@@ -127,26 +127,19 @@ controller:
   # Allowed values: map of key-value pairs
   # Default value: None
   # Examples:
-  # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
-  #  node-role.kubernetes.io/master
```

Comment:
Maybe I missed something, but why are we undoing this change?

Reply:
for labels: we can use control-plane, as all k8s versions we support have that label (was introduced in k8s 1.20)